easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.10.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,39 @@
+name = 'TensorFlow'
+version = '1.10.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_remove-msse-hardcoding.patch',
+    '%(name)s-%(version)s_dont_expand_cuda_cudnn_path.patch',
+]
+checksums = [
+    'ee9cb98d9e0d8106f2f4ed52a38fe89399324af303e1401567e5b64a9f86744b',  # v1.10.0.tar.gz
+    # TensorFlow-1.10.0_remove-msse-hardcoding.patch
+    'd7a37d6d2be6a35b262a27ad77d3c8a4746cc7b3ae8f93b979afd947821fd1a2',
+    # TensorFlow-1.10.0_dont_expand_cuda_cudnn_path.patch
+    'a32dc8b5601e588e29464847daeccb57ce5943e79642cf237fbd8b6ff25f40c0',
+]
+
+builddependencies = [
+    ('Bazel', '0.16.0'),
+    ('wheel', '0.31.1', versionsuffix),
+]
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('cuDNN', '7.1.4.18'),
+]
+
+preconfigopts = 'export CC_OPT_FLAGS="$CC_OPT_FLAGS -mno-avx512f" && '
```

Comment:
@akesandgren This can be removed if the issues on AVX512 are indeed fixed

Reply:
Wanted to verify the AVX512 problem using an intel based build but boringssl/grpc/libssl are causing problems there. Will try a bit more to get that build to work first.